pages/Documents/DeveloperTools/LivePersonFunctions/DevelopingwithFaaS/events-templates.md

```diff
@@ -43,7 +43,13 @@ Currently the following events are supported:
 
 
 Each of these events also has an event specific input (payload) associated with it, that the triggering system will provide at runtime. When using the [Invoke](function-as-a-service-deploying-functions.html#testing-your-function) screen, this payload is shown on the left-hand side. During development, the event payload of the `lambda` is visible on the right-hand side at the sidebar-tab. See the [Payload](function-as-a-service-developing-with-faas-overview.html#editor-sidebar) section for more information.
+#### Support for limiting invocations by skills
 
+Certain events can be more restricted to only invoke a lambda on certain skills. This not only allows you to be more specific in your lambda invocations saving you some lines of code but it also means that it reduces the number of meaningless invocations.
+
+<img src="img/faas-select-skill.png" alt="Functions select skill" style="width:100%;"/>
+
+To specify to which skill the lambda should react simply select the skills during the creation/editing process of your lambda. By default, the lambda will react to all skills. If the lambda should only react to specific skills then there is a maximum of <b>10</b> skills that can be selected at any given time.
```

Comment:
To specify to which skill the lambda should react, simply select the wanted skill during the creation/editing process. By default, the lambda will react to all skills. 
Please be aware that you can overall only select a maximum of <b>10</b> skills per lambda.

Reply:
We should also include here the hint, that changing a skill later (won't change the state). And that the change should be reflected within 5min's on productive functions.